### PR TITLE
Lightning talk booking improvements

### PR DESCRIPTION
**Pull request by vizvamitra**
Three main changes in lightning talk slots booking:

  - Allow only one lightning talk slot per user
  - Implement booking confirmation
  - Implement booking cancellation

Also some refactoring

[video](https://drive.google.com/open?id=19WWRdYwNe51DG9wPtcJWIlWw3_dBWwLC)

### CODE REVIEW

File: app/commands/lightning_talks/book.rb [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+          parse_mode: :markdown,
+          reply_markup: inline_keyboard(*available_slot_buttons(username))
+        )
+      end
+
+      def handle_callback(callback, args)
+        id = args.fetch('id') { raise(FallbackError) }
+        slot = schedule_lightning_repo.by_id(id) || raise(FallbackError)
+        username = username_for(callback.from)
+        confirmed = args.fetch('confirm', false)
```
Comment (we138) [inline app/commands/lightning_talks/book.rb:27]: maybe this will be more clearly (imho)

```
if args['confirm'].nil?
 ...
```

**Comment by we138** [comment]
good work!

**Review by we138 — APPROVED** [review]

File: app/commands/lightning_talks/book.rb (hunk 1 above)
Comment (vizvamitra) [reply app/commands/lightning_talks/book.rb:27]: I  wanted to separate getting command's context from logics and keep the code in the `if` part at a single level of abstraction

Comment (we138) [reply app/commands/lightning_talks/book.rb:27]: kk

**Pull request merged**